pkg/kube-metrics/metrics.go

```diff
@@ -111,3 +122,22 @@ func GetNamespacesForMetrics(operatorNs string) ([]string, error) {
 	}
 	return ns, nil
 }
+
+func isNamespaced(gvk schema.GroupVersionKind, cfg *rest.Config) (bool, error) {
+	discoveryClient, err := discovery.NewDiscoveryClientForConfig(cfg)
+	if err != nil {
+		log.Error(err, "Unable to get discovery client")
+		return false, err
+	}
+	resourceList, err := discoveryClient.ServerResourcesForGroupVersion(gvk.GroupVersion().String())
+	if err != nil {
+		log.Error(err, "Unable to get resource list for", "apiversion", gvk.GroupVersion().String())
+		return false, err
+	}
+	for _, apiResource := range resourceList.APIResources {
+		if apiResource.Kind == gvk.Kind {
+			return apiResource.Namespaced, nil
+		}
+	}
+	return false, errors.New("Unable to find type: " + gvk.String() + " in server")
```

Comment:
```suggestion
	return false, errors.New("unable to find type: " + gvk.String() + " in server")
```

This one need to be lower case. See:

/home/travis/gopath/src/github.com/operator-framework/operator-sdk/pkg/kube-metrics/metrics.go:142:	return false, errors.New("Unable to find type: " + gvk.String() + " in server")
Makefile:209: recipe for target 'test-sanity' failed